docs/install/traditional.rst

```diff
@@ -37,12 +99,12 @@ Install Drupal and Tripal
       
       The ``web`` directory is the actual webroot for Drupal. This should be the directory that is served by your webserver. **The two composer files and the** ``vendor`` **directory should not be publicly accessible.**
 
-2. Install Drush as well, also with composer, ensuring that you are within your new ``tripal4`` directory:
+2. Install Drush and other required modules, also with composer, ensuring that you are within your new ``tripal4`` directory:
 
     .. code-block:: shell
 
       cd /var/www/tripal4/
-      composer require drush/drush
+      composer require drush/drush drupal/field_group drupal/field_group_table
 
 3. Clone the Tripal repository in your ``web/modules`` directory.
```

Comment:
There are actually two options here depending on the purpose of your site. If you are a developer then cloning it as you've done here is the right way to go. However, if not then you should actually use composer to get tripal in the same way you did in step 2.

This is the Tripal packagist page: https://packagist.org/packages/tripal/tripal

To just use the most recent stable version of tripal you use `composer require tripal/tripal`. 

To install a specific version you find the tag in the right column and use it like this: `composer require tripal/tripal:4.0-alpha2`

To install the most recent development version you use `composer require tripal/tripal:4.x-dev`

Reply:
Nice to know. I will put most of this in the docs! I left the packagist link out since it may be unneeded for this step-by-step instruction, but can add if if you like.

I notice the packagist page says "Requires  php: ^8.0" - something else to fix 😮

Reply:
Perfect! And Sukha is working on the version update 👍 Good catch!